### DISCUSSION TITLE
Welcome to s3m Discussions!

### DISCUSSION THREAD

**Discussion by nbari** [s3m/s3m#23] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JeffAnton** [comment]
One machine I've installed 0.9.0 on is failing with an error.
% ./s3m show
error: invalid value '18446744073709551615' for '--number <number>': number too large to fit in target type
and what's interesting is the the help information shows that same wrong number as the default.
This is FreeBSD 13.2 pkg. Three other machines with the same pkg installed all work fine.
I can use s3m on the failing machine if I pass --number 1 to all commands. It's annoying and a little concerning that something in the environment is changing a default. That should be documented.
I've done sha1 of the binary on all machines and they all match so the executable seems ok.
Any suggestions?

**Reply by nbari** [reply]
https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=278960

**Comment by nbari** [comment]
Hi @JeffAnton I think there is a bug when trying to find the available [number of cores](https://github.com/s3m/s3m/blob/main/src/cli/commands/mod.rs#L75), can you please share the output of `s3m --help`

And in the VM that is failing share the output of `sysctl -a | egrep -i 'hw.machine|hw.model|hw.ncpu'`

**Comment by nbari** [comment]
Hi @JeffAnton despite that , many thanks for the input, I am working now on a fix to check the data types  ranges no mater if the num_cpu is failing
